app/controllers/rooms_controller.rb

```diff
@@ -115,41 +115,65 @@ def game_player_count_post
     count = params[:player_count].to_i
     @room.player_count = count
     @room.save
+    # Delete players that no longer fit in this game
+    ActiveRecord::Base.connection.execute("DELETE FROM room_players where room_id = #{@room.id} AND player_number > #{(count / 2).to_i}")
```

Comment:
So, I think you can do this like so:
```ruby
RoomPlayers.all.where('room_id = ? AND player_number > ?", @room.id, (count / 2).to_i).delete_all 
```
and that'll get you away from string injections and what not.